packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -34,6 +34,7 @@ type TAutocompleteProps = {
     value: string;
     onSearch?: (value: string, items: TItem[]) => [];
     data_testid: string;
+    readOnly: boolean;
```

Comment:
@maryia-matskevich-deriv this property, since we are adding it, doesn't it need to be optional?

Reply:
✅done